formatTest/unit_tests/expected_output/jsx.re

```diff
@@ -623,21 +623,17 @@ ReasonReact.(<> {string("Test")} </>);
 </div>;
 
 <div>
-  {
-    let left = limit->Int.toString;
-    {j|$left characters left|j}->React.string;
-  }
+  {let left = limit->Int.toString;
+   {j|$left characters left|j}->React.string}
 </div>;
 
 <View style=styles##backgroundImageWrapper>
-  {
-    let uri = "/images/header-background.png";
-    <Image
-      resizeMode=`contain
-      style=styles##backgroundImage
-      uri
-    />;
-  }
+  {let uri = "/images/header-background.png";
```

Comment:
We don't have this form of wrapping anywhere else (not intentionally at least). Is there a reason why this isn't breaking into multiple lines?
```
{
  let uri = 
  ...
}
```

Reply:
This has been here for a while, e.g. https://github.com/facebook/reason/pull/2208.

We did it initially to save space on JSX patterns like:

```js
<div>
  {
    React.string("foo")
  }
```